Codemine/Extensions/UIImage+Utilities.swift

```diff
@@ -24,7 +24,7 @@ public extension UIImage {
      
      - Returns: A 'UIImage' with the specified color, size and corner radius.
      */
-    public class func imageFromColor(_ color: UIColor, size: CGSize, cornerRadius: CGFloat) -> UIImage? {
+    public class func from(color: UIColor, size: CGSize, cornerRadius: CGFloat) -> UIImage? {
```

Comment:
Is it too crazy to make this a convenience initialiser instead of a class function? Think it would make more sense.